tests/template_tests/syntax_tests/test_cache.py

```diff
@@ -122,6 +122,14 @@ def test_cache18(self):
         output = self.engine.render_to_string('cache18')
         self.assertEqual(output, 'cache18')
 
+    @setup({'cache19': '{% load cache %}{% cache None test %}cache19{% endcache %}'})
+    def test_cache19(self):
+        """
+        Allow cache timeout to be None (#27882).
```

Comment:
Ticket reference isn't needed. The docstring could be omitted in place of a meaningful test name, e.g. `test_timeout_none`. Could set test be added to check that `cache.set()` is called with `None`, i.e. that the value is actually cached?